modules/meta.analysis/R/approx.posterior.R

```diff
@@ -43,7 +43,9 @@ approx.posterior <- function(trait.mcmc, priors, trait.data=NULL, outdir=NULL, f
     
     dat    <- trait.mcmc[[trait]]
     vname  <- colnames(dat[[1]])
-    dat    <- as.matrix(dat)[,'beta.o']
+    if('beta.o' %in% vname) {
```

Comment:
>At least we should update the documentation at ##' @param trait.mcmc to indicate it can be either a list or vector.
>
>On the other hand, it might be more straightforward to use the trait.samples list (which is saved in samples.Rdata by get.paramter.samples) here. (would it be more straightforward? Where are the PDA results stored?)

(I assume you meant to post this here, @dlebauer )

When the PDA code calls `approx.posterior`, `trait.mcmc` is a list too. The PDA parameter samples are actually a matrix throughout most of the code, but I coerce them to a list just to send them into `approx.posterior`.

If the meta analysis code called `approx.posterior` on `trait.samples` instead, that would be more like what the PDA code does and so yes, does seem more straightforward. The MA samples would still be in a list with parameter names naming the elements, but with each element being simply a vector of parameter values, so need to deal with finding 'beta.o' at all. 

The PDA samples are saved to .../pft/*/mcmc.pdaENSEMBLEID.Rdata.